download/src/index.js

```diff
@@ -9,6 +9,7 @@ import fetch from 'node-fetch';
 import retry from 'async-retry';
 import which from 'which-promise';
 import readPkg from 'read-pkg';
+import HttpsProxyAgent from 'https-proxy-agent';
```

Comment:
It would be better to use the [`proxy-agent`](https://npmjs.org/proxy-agent) module for this, so that SOCKS proxies would also be supported.